### PR TITLE
change deployment node instances to *string

### PR DESCRIPTION
**Pull request by beffge**
Fix for https://github.com/goadesign/model/issues/333

I think I found all instances of `instances` and resolved all errors when `make test` runs 😇 

Please be patient with me, I'm no professional programmer.

### CODE REVIEW

**Review by raphael — COMMENTED** [review]
This looks really good! Left just one small change request - thank you!

File: dsl/deployment.go [hunk 1]
```diff
@@ -293,18 +293,20 @@ func ContainerInstance(container interface{}, dsl ...func()) *expr.ContainerInst
 //
 // Instances must appear in a DeploymentNode expression.
 //
-// Instances accepts a single argument which is the number.
+// Instances accepts a single argument which is the string.
```
[the remark this answers is not in the document]
Comment (beffge) [reply dsl/deployment.go]: deal 😉

**Comment by beffge** [comment]
> This looks really good! Left just one small change request - thank you!

Thanks!

**Comment by beffge** [comment]
image

Would you be so kind?

**Comment by raphael** [comment]
Thank you!

**Pull request merged**